thriftbp/errors.go

```diff
@@ -32,6 +32,15 @@ type baseplateError interface {
 	GetDetails() map[string]string
 }
 
+// We create this interface instead of using baseplateError because we want it to always match
+// regardless of the version of the thrift file the user has put in their repository.
+type baseplateErrorCode interface {
```

Comment:
It's not an error code, it's a thing that gives you codes, so `baseplateErrorCoder` (reader, writer, etc)

Reply:
renamed (again)